tests/compare_tables/2columns_most_common.txt

```diff
@@ -1,15 +1,15 @@
- EXTENSION       | FREQ. 
------------------+-------
- TXT             |     3 
- GZ              |     3 
- MD              |     2 
- [no extension]  |     2 
- PY              |     2 
- HTML            |     1 
- JSON            |     1 
- CSS             |     1 
- WOFF            |     1 
------------------+-------
- TOTAL:          |    16 
------------------+-------
+ EXTENSION      | FREQ. 
+----------------+----
```

Comment:
There is something wrong here. The column width shouldn't be smaller than "FREQ.".

```
  EXTENSION       | FREQ.
 -----------------+-------
```

Reply:
the word "FREQ." is longer than the digits

Reply:
Exactly. There used to be somewhere a constraint to make that column fit the word "FREQ.". In the version I still have in my system, there are always 7 dashes below that word (i.e. column size == 5 plus spaces).

Reply:
But it's a bit strange, the changes you made don't seem to be related to this issue. Maybe it was me. I will do some tests.

Reply:
```
$ count-files ~/Dropbox/dev/Count-files/tests/

                                                                                           
 EXTENSION      | FREQ. 
----------------+-------
 TXT            |    15 
 PY             |    14 
 [no extension] |     5 
 PYC            |     3 
 GZ             |     3 
 MD             |     2 
 XLSX           |     2 
 HTML           |     1 
 JSON           |     1 
 CSS            |     1 
 WOFF           |     1 
 DAT            |     1 
----------------+-------
 TOTAL:         |    49 
----------------+-------
```

Seems to be working properly, my latest version.

Reply:
? sep_right = (freq_col_width + 2) * '-' if freq_col_width > 5 else 7 * '-'

Reply:
in last version: `{'FREQ.'.ljust(freq_col_width)}`
`.ljust`

Reply:
it looks like this now:
![last](https://user-images.githubusercontent.com/23127253/44929160-addad480-ad62-11e8-8f9b-d461d9d908c7.png)

Reply:
I have this in line 42 (viewing_modes.py):
```
    sep_right = (freq_col_width + 2) * '-'
```

Reply:
so: (freq_col_width + 2) -> (2 + 2) -> '----'

Reply:
Nope. On line 33, we have this:

```
freq_col_width = min(DEFAULT_FREQ_COL_WIDTH, len(str(total_occurences)))
```
The constant DEFAULT_FREQ_COL_WIDTH is set to 5 in settings.py, because that's the length of the string "FREQ.". That column should never have a smaller width for that reason.

Reply:
Let me check here if I am not running the old Test PyPI version :-)

Reply:
Damn!... I guess it was that. Now I am getting the same result as you. Sorry! That `min()` I put there is obviously wrong. Can you please fix it for me?